CHANGELOG.asciidoc

```diff
@@ -1,4 +1,5 @@
 include::./changelogs/head.asciidoc[]
+include::./changelogs/8.8.asciidoc[]
```

Comment:
**question**: is this missing ? Am I missing something ?

Reply:
Nice catch, deleted head.asciidoc from this asciidoc as well.

Reply:
yes 8.8 was missing in PR, only exists in unstaged area locally :facepalm: